include/seqan3/io/detail/magic_header.hpp

```diff
@@ -13,9 +13,11 @@
 #pragma once
 
 #include <array>
+#include <string>
+#include <vector>
```

Comment:
in the node you added, you do not use `vector` or `string` ?

Reply:
Yes, but I cleaned up the includes and string and vector are used inside of the file at other places.